src/api/API.ts

```diff
@@ -0,0 +1,46 @@
+import {
+  Project,
+  Customer,
+  CustomerSignInResult,
+  CustomerDraft,
+} from '@commercetools/platform-sdk/dist/declarations/src/generated';
+import { ByProjectKeyRequestBuilder } from '@commercetools/platform-sdk/dist/declarations/src/generated/client/by-project-key-request-builder';
+
+export class API {
```

Comment:
Class components. It would be better to use functions. We can add it to things todo for our improvements